tools/flang2/flang2exe/llassem.cpp

```diff
@@ -1164,8 +1164,13 @@ assemble_end(void)
       tdefsym = find_ag(gname);
       typed = AG_TYPENAME(tdefsym);
       fprintf(ASMFIL, "%%struct%s = type < { %s } > \n", name, typed);
+#if defined (_WIN64)
+      fprintf(ASMFIL, "@%s = %s global %%struct%s ", name,
+              AG_ISMOD(gblsym) ? "external dllimport" : "common", name);
```

Comment:
Is this generally correct for all compilation units, even for code intended for static archives?